### PR TITLE
Remove unnecessary rounding from f32xN::trunc.

### PR DESCRIPTION
**Pull request by RazrFalcon**

### CODE REVIEW

**Review by Lokathor — APPROVED** [review]

**Comment by RazrFalcon** [comment]
Fixed.

File: src/f32x4_.rs [hunk 1]
```diff
@@ -518,33 +518,23 @@ impl f32x4 {
...
-    all(
-      any(target_arch = "x86", target_arch = "x86_64"),
-      target_feature = "sse"
-    ),
-    all(
-      not(any(target_arch = "x86", target_arch = "x86_64")),
-      feature = "std"
-    ),
-  ))]
+  #[cfg(feature="std")]
```
Comment (Lokathor) [inline src/f32x4_.rs:521]: This needs to allow for no_std with sse enabled:
`#[cfg(any(target_feature="sse", feature="std"))]`

Comment (Lokathor) [reply src/f32x4_.rs:521]: And the similar configs

**Comment by RazrFalcon** [comment]
Fixed? I'm not sure anymore...

**Comment by Lokathor** [comment]
seems good!

**Pull request merged**